### PR TITLE
updated  first row

### PR DESCRIPTION
**Pull request by rohitsainik**
added flexible as text was showing error

### CODE REVIEW

**Comment by itzmeanjan** [comment]
Hello @rohitsainik , thanks for the PR, it'll be reviewed soon :)

File: lib/SensorInfoHolder.dart [hunk 1]
```diff
@@ -64,15 +64,21 @@ class SensorInfoHolder {
       Row(
         mainAxisAlignment: MainAxisAlignment.spaceBetween,
         children: <Widget>[
-          Text(
```
Comment (itzmeanjan) [inline lib/SensorInfoHolder.dart:67]: Was text getting overflown ?

**Review by itzmeanjan — COMMENTED** [review]
Ok, I'll merge this into master. Thanks for PR :)

**Comment by rohitsainik** [comment]
Appreciated 👍 @itzmeanjan

**Pull request closed without merging**